tests/test_hs_restclient.py

```diff
@@ -470,5 +470,25 @@ def test_resource_list_by_bounding_box(self):
         for (i, r) in enumerate(res_list):
             self.assertEquals(True, True)
 
+
+class TestReferencedFile(unittest.TestCase):
+    @with_httmock(mocks.hydroshare.resourceCreateReferenceURL_post)
+    def test_referenced_create(self):
+        hs = HydroShare(prompt_auth=False)
+
+        response = hs.createReferencedFile(pid='511debf8858a4ea081f78d66870da76c', path='data/contents', name='file.url',
+                                           ref_url='https://www.hydroshare.org')
+
+        self.assertEqual(response['status'], 'success')
+
+    @with_httmock(mocks.hydroshare.resourceUpdateReferenceURL_post)
+    def test_referenced_update(self):
+        hs = HydroShare(prompt_auth=False)
+
+        response = hs.updateReferencedFile(pid='511debf8858a4ea081f78d66870da76c', path='data/contents', name='file.url',
+                                           ref_url='https://www.hydroshare.org')
```

Comment:
Should we use a different value of the ref_url than what has been used in creating the referenced file?